ingesters/winevents/main.go

```diff
@@ -189,22 +188,26 @@ type levelLogger struct {
 }
 
 // levelLogger implements the log.LevelRelay interface
-func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg []byte) error {
+func (l levelLogger) WriteLog(lvl log.Level, ts time.Time, msg, rawMsg string) error {
 	switch lvl {
 	case log.DEBUG:
 		if debugOn {
-			fmt.Fprintln(os.Stdout, string(msg))
+			fmt.Fprintln(os.Stdout, rawMsg)
 		}
 	case log.INFO:
-		return l.elog.Info(1, string(msg))
+		return l.elog.Info(1, rawMsg)
 	case log.WARN:
-		return l.elog.Warning(1, string(msg))
+		return l.elog.Warning(1, rawMsg)
 	case log.ERROR:
 		fallthrough
 	case log.CRITICAL:
 		fallthrough
 	case log.FATAL:
-		return l.elog.Error(1, string(msg))
+		return l.elog.Error(1, rawMsg)
```

Comment:
Does the Error() method exit? Do we have a Fatal() method? It doesn't look like it from the fallthroughs above but I thought I'd ask.

Reply:
the `l.elog.Error` ?

Reply:
yes, we have a fatal method, and it does exit.  This switch just controls what makes it "out", the caller then terminates